source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -133,6 +133,13 @@ Read Preference
 connecting to the source and destination clusters. For more information,
 see :ref:`connections-read-preference`. 
 
+Rewrites
```

Comment:
[question] Is this the right title? Is this what would help a user understand what is being rewritten?

Reply:
Thank you for taking a look at this, @mdb-ashley !

I specified the title a bit – let me know if you think it's clearer! Also, I was a little unsure if this was the write section to put this information in. Does it seem right to you, or do you think there's a more appropriate place for it?